More reader-friendly structure in README (#148)
abelsiqueira/COPIERTemplate.jl: README.md

</div>

## What does `COPIERTemplate` do?

Creating `Julia` packages involve the creation and edition of many tiny files.
Wouldn't it be great to automate this?

This is exactly what `COPIERTemplate` does.

### FAQ

- How is `COPIERTemplate` different from `PkgTemplates`?
  - it can be applied to existing packages
  - it invites to follow some (opinionated) best practices

>   - it is automatically reapplied through Pull Requests made by the `Copier.yml` workflow

Reviewer: ```suggestion
  - it can be reapplied to acquire updates made to the template
  - it is automatically reapplied through Pull Requests made by the `Copier.yml` workflow (Work in progress)
```
Updating my list and marking that feature as WIP.

### Architecture

Under the hood, `COPIERTemplate` is no more and no less than:

- a [copier](https://copier.readthedocs.io) template/skeleton for Julia packages (see folder [template](template)); and
- a package that wraps `copier` in Julia using `PythonCall`.

## Quickstart

Install `COPIERTemplate` in your chosen environment (we recommend base):

```julia-repl

> julia> add COPIERTemplate

Reviewer: ```suggestion
pkg> add COPIERTemplate
```

Author: Perhaps even better for newcomers:

```suggestion
julia> using Pkg
julia> Pkg.add("COPIERTemplate")
```

</div>

## What does `COPIERTemplate` do?

Creating `Julia` packages involve the creation and edition of many tiny files.
Wouldn't it be great to automate this?

This is exactly what `COPIERTemplate` does.

### FAQ

- How is `COPIERTemplate` different from `PkgTemplates`?
  - it can be applied to existing packages
  - it invites to follow some (opinionated) best practices
  - it can be reapplied to acquire updates made to the template
  - it is automatically reapplied through Pull Requests made by the `Copier.yml` workflow (Work in progress)

### Architecture

Under the hood, `COPIERTemplate` is no more and no less than:

- a [copier](https://copier.readthedocs.io) template/skeleton for Julia packages (see folder [template](template)); and
- a package that wraps `copier` in Julia using `PythonCall`.

## Quickstart

> Install `COPIERTemplate` in your chosen environment (we recommend base):
> 
> ```julia-repl
> julia> using Pkg
> julia> Pkg.add("COPIERTemplate")
> ```

Reviewer: ```suggestion
Install `COPIERTemplate` in your chosen environment (we recommend globally) by entering `pkg` mode by pressing `]` and then:

```julia-repl
pkg> add COPIERTemplate
```
```
